### PR TITLE
Revert "Add date opened to every update block"

### PR DESCRIPTION
**Pull request by czue**
Reverts dimagi/commcare-hq#12344

want to see if this is the cause of http://manage.dimagi.com/default.asp?232602

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/case/mock.py [hunk 1]
```diff
@@ -117,7 +115,9 @@ def __init__(self,
             case_name = "" if case_name is CaseBlock.undefined else case_name
             owner_id = "" if owner_id is CaseBlock.undefined else owner_id
         self['update'] = update
-
+        self['update'].update({
+            'date_opened':                  date_opened
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/mock.py]: E241 multiple spaces after ':'

File: corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py [hunk 2]
```diff
@@ -74,7 +73,7 @@ def test_couch_action_equality(self):
         _post_util(case_id=case_id, p1='p1', p2='p2')
 
         case = CommCareCase.get(case_id)
-        self.assertEqual(3, len(case.actions))  # (1) create & (2) update date opened (3) update properties
+        self.assertEqual(2, len(case.actions)) # create + update
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py]: E261 at least two spaces before inline comment

File: corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py [hunk 3]
```diff
@@ -107,13 +106,11 @@ def test_couch_soft_rebuild(self):
         self.assertEqual(case.p1, 'p1-1') # original
         self.assertEqual(case.p2, 'p2-2') # updated
         self.assertEqual(case.p3, 'p3-2') # new
-        self.assertEqual(4, len(case.actions)) # create + update (2 actions) + 2 updates
-        a0 = case.actions[1]
-        self.assertEqual(a0.updated_known_properties['opened_on'], case.opened_on)
-        a1 = case.actions[2]
+        self.assertEqual(3, len(case.actions)) # create + 2 updates
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py]: E261 at least two spaces before inline comment

**Comment by czue** [comment]
@proteusvacuum this broke all kinds of stuff on mobile. we should better understand / test before rolling it out again

**Comment by proteusvacuum** [comment]
😢 sorry! I had talked to Clayton about this, and it seemed like a safe change to make. I should probably have tested on mobile first before pushing this though. I'll write up a retro.

**Pull request merged**